### PR TITLE
docs(CONTRIBUTING): Apply #6878 to the rest of the CONTRIBUTING files

### PR DESCRIPTION
**Pull request by LuigiImVector**
## What does this PR do?
Improve repo

### Description

Apply to the rest of the CONTRIBUTING files the changes made on #6878 about the alphabetical order.
Inspired by @davorpa (#6824).

## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [X] Put lists in alphabetical order, correct spacing.
- [X] Add needed indications (PDF, access notes, under construction)
- [X] Translated into English
- [X] Translated into Deutch: @lupinitylabs
- [ ] Translated into Greek
- [ ] Translated into Persian
- [ ] Translated into Filipino
- [X] Translated into French: @lorrding
- [ ] Translated into Indonesian
- [X] Translated into Italian: @LuigiImVector
- [ ] Translated into Korean
- [ ] Translated into Portuguese
- [ ] Translated into Russian
- [X] Translated into Spanish: @davorpa 
- [ ] Translated into Vietnamese
- [ ] Translated into Chinese (simplified)
- [X] Translated into Chinese (traditional): @AlanSyue

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by davorpa — COMMENTED** [review]
I provide the Spanish translation 🎉

File: docs/CONTRIBUTING-es.md [hunk 1]
```diff
@@ -163,6 +170,7 @@ Nuestros listados proporcionan un conjunto mínimo de metadatos: títulos, URL,
 
 - Sin títulos inventados: Intentamos tomar el texto de los propios recursos; se advierte a los colaboradores que, si puede evitarse, no inventen títulos ni los utilicen editorialmente. Una excepción es para obras más antiguas: si son principalmente de interés histórico, un año entre paréntesis adjunto al título ayuda a los usuarios a saber si estos son de interés.
 - Sin títulos TODO EN MAYÚSCULAS: Por lo general, es apropiado tener cada primera letra de palabra en mayúsculas, pero en caso de duda, use siempre el estilo tal y como viene en la fuente original.
+- No emojis.
```
[the remark this answers is not in the document]
Comment (LuigiImVector) [reply docs/CONTRIBUTING-es.md]: Thanks <3

**Comment by LuigiImVector** [comment]
@AlanSyue, @bluetata, @tronghieu60s, @ricardokojo, @andylang8445, @amirmojiry, @lupinitylabs, @Existential-Kernel, @kang8 sorry for the inconvenience, it would be great if you would help translate these parts of files.

**Comment by AlanSyue** [comment]
@LuigiImVector 
Sure! I can help translate to traditional Chinese.

**Comment by amirmojiry** [comment]
I can work on the Persian (fa_IR) translation.

File: docs/CONTRIBUTING-fr.md [hunk 2]
```diff
@@ -148,6 +155,7 @@ Nos listes fournissent un ensemble minimal de métadonnées : titres, URL, créa
 
 - Pas de titres inventés. Nous essayons de prendre les titres des ressources elles-mêmes ; les contributeurs sont avertis de ne pas inventer de titres ou de ne pas les utiliser éditorialement si cela peut être évité. Une exception est pour les œuvres plus anciennes; s'ils présentent principalement un intérêt historique, une année entre parenthèses ajoutée au titre aide les utilisateurs à savoir s'ils présentent un intérêt.
 - Pas de titres TOUTES EN MAJUSCULES. Habituellement, la casse du titre est appropriée, mais en cas de doute, utilisez la majuscule de la source
+- No emojis.
```
Comment (lorrding) [inline docs/CONTRIBUTING-fr.md:158]:
```suggestion
- N'utilisez pas d'émoticônes.
```

**Review by lorrding — CHANGES_REQUESTED** [review]
Hi, I provided French translations :)
Tell me if something feels off.

File: docs/CONTRIBUTING-fr.md [hunk 3]
```diff
@@ -133,6 +133,13 @@ Exemple:
     BIEN   : * [Sera bientôt un livre génial](http://exemple.com/livre2.html) - John Doe (HTML) (:construction: *in process*)
     ```
 
+### Alphabetical order
+
+- When there are multiple titles beginning with the same letter order them by the second, and so on. For example: `aa` comes before `ab`.
+- `one two` comes before `onetwo`
+
+If you see a misplaced link, check the linter error message to know which lines should be swapped.
+
```
[the remark this answers is not in the document]
Comment (LuigiImVector) [reply docs/CONTRIBUTING-fr.md:142]: In the title insert the anchor link: `### <a id="alphabetical-order"</a> Ordre...`

[the remark this answers is not in the document]
Comment (lorrding) [reply docs/CONTRIBUTING-fr.md:142]: It's updated 👍

[the remark this answers is not in the document]
Comment (LuigiImVector) [reply docs/CONTRIBUTING-fr.md:142]: thank you for your contribution❤

**Comment by LuigiImVector** [comment]
> I can work on the Persian (fa_IR) translation.

let's wait for his translation first

**Comment by LuigiImVector** [comment]
No one else has replied so I think it's ready to be merged

**Pull request merged**